### ISSUE TITLE
[ACTIVE_ARBITRATION_WIN] SWARM-1550 — PR1372 G06 exact-byte execution repair + real replay

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1550]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: SWARM-1550
STATUS: ACTIVE_STALE_TAKEOVER_PENDING_POSTCLAIM_DIRECT_ARBITRATION
LOCAL_FREE_ONLY: true
PREFERRED_LANE: #7 D06 — Evaluation / Benchmarks / Stage Gates
PREFERRED_WORK_KIND: REPRODUCIBILITY_RELEASE
SELECTED_WORK_KIND: SAME_LINEAGE_STALE_TAKEOVER / DEFECT_REPAIR / REAL_LOCAL_FREE_EXECUTION / REPRODUCIBILITY_CLOSURE
SWARM_LANE_KEY: D03|CURRENT-SURVIVOR|G06-PRIVACY-EXECUTION-AUTHORITY|PAYLOAD-BOUND-RECEIPT-V1
TARGET_PR: #1372
TARGET_BRANCH: d03/g06-privacy-execution-authority-1365-v1
TARGET_REJECTED_HEAD: 9562aa23e32240b3409fa98a0937b411cce1e22e
PREDECESSOR_PRODUCT_OWNER: #1489
PREDECESSOR_LAST_ISSUE_UPDATE_UTC: 2026-09-12T06:01:13Z
PREDECESSOR_DECLARED_LEASE: six hours from issue update
PREDECESSOR_LEASE_EXPIRED_UTC: 2026-09-12T12:01:13Z
LIVE_MAIN_AT_CLAIM: 4b8b2392ff3713c1b48e633b1b05a291ccb0c40c
CURRENT_INTEGRATION_OWNER: #1546 or lawful successor
CLAIMED_AT_UTC: THIS_ISSUE_UPDATED_AT_TIMESTAMP_IS AUTHORITY
LEASE: six hours from this issue update unless released sooner

STALE-TAKEOVER BASIS:
1. #1489 declared an evidence-bound six-hour Product lease from its issue update. Its current issue `updated_at` is `2026-09-12T06:01:13Z`; therefore the lease expired at `2026-09-12T12:01:13Z`.
2. This claim is after that boundary. Fresh late-bind still shows PR #1372 OPEN / non-draft / mergeable on exact rejected head `9562aa23e32240b3409fa98a0937b411cce1e22e`; no Product head has moved since the independent HIGH.
3. Exact real LOCAL_FREE replay on this head genuinely executed the 272-record / 5,921,963-byte survivor graph twice with byte-identical text-free output, but #1492 terminally rejected scientific authority with `AUD1492-G06-002 HIGH`: the implementation pins `privacy_filter_v3.py` source bytes yet executes mutable already-imported callables whose writable `__module__` can be forged.
4. #1498 supplied a narrow semantics-preserving repair: read canonical privacy source bytes exactly once, verify their exact Git blob, compile/execute those SAME verified bytes in a fresh private module namespace, and obtain both `hash_safe_scan` and `policy_manifest` from that namespace; poisoned live canonical module objects/helpers/globals must not affect execution.
5. G05 is now genuinely integrated on current main. Downstream PR #1041 is under independent audit #1542 and explicitly blocks on nonterminal G06 plus physical redaction/exclusion materialization; #1545/PR #1549 owns the post-G05/G06 materialization Product. Therefore terminalizing G06 is an immediate current-corpus gate, not distant infrastructure.
6. Preclaim semantic search found no newer same-semantic Product successor after #1489 expiry. Mandatory postclaim direct collection arbitration now decides authority before any Product mutation.

CANDIDATE COMPARISON:
1. PR #1372 G06 same-lineage stale repair + real replay — SELECTED: expired owner, exact HIGH and minimum repair known, real payload already reaches the gate, downstream materialization waits on it.
2. PR #1041 audit — YIELD to #1542.
3. PR #1410 audit — YIELD to #1527.
4. PR #1543 Trainer resume — YIELD to active #1539; its current exact head is CI-red and still under live Product lease.
5. Integration — YIELD to #1546.
6. Start learned-20M training now — REJECT solely because `AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0` and downstream split/packing/unique-loss/tokenizer/runtime gates are nonterminal; switch immediately when positive/terminal.
7. New source family/readiness framework — REJECT while this stale real-data gate is directly repairable.

OWNED SAME-LINEAGE SURFACE IF POSTCLAIM ARBITRATION WINS:
- continue existing PR #1372 and existing branch ONLY; no new architecture or successor PR;
- repair `src/twelve_six/data/privacy_execution_authority.py::_privacy_binding` so execution is bound to the exact verified canonical source bytes rather than mutable live imported module objects;
- resolve expected canonical source path without trusting a live imported module object;
- read source bytes once, recompute exact Git blob and require `bcc5938395724f6728ab212f98b39f2334b0f37d`;
- compile/execute those SAME bytes in a fresh private module namespace, temporarily registered only under a unique private name when needed for dataclass/annotation machinery, then removed; never consume pre-poisoned canonical `sys.modules` behavior;
- obtain `hash_safe_scan` and `policy_manifest` from the fresh verified-byte namespace and preserve canonical metadata/policy identity;
- add adversarial regressions for forged `hash_safe_scan.__module__`, forged `policy_manifest.__module__`, and mutated live helper/global (`detect` / `DETECTOR_ACTIONS`) while canonical source bytes remain unchanged; repaired build+verify must ignore poisoned live state or fail closed before execution;
- preserve the already-correct no-LF DATA526 inventory-root repair and all zero-credit truth boundaries;
- focused/adversarial LOCAL_FREE tests + genuine current-survivor replay; real negative result is acceptable and must be retained honestly;
- if repaired real replay succeeds, require repeat byte identity, retain compact text-free evidence, remove temporary execution carrier, obtain exact-final-head shared CI, then release for fresh DIFFERENT-worker audit; no self-certification and no merge.

PACKAGE_DIMENSIONS:
- implementation_or_primary_research
- focused_tests
- adversarial_or_negative_tests
- real_local_free_execution
- machine_readable_evidence_or_validator
- reproducibility_or_recovery_proof
- live_authority_binding
- end_to_end_or_integration_proof
- operator_handoff

AVOID:
- no mutation of privacy detector vocabulary/actions/thresholds;
- no G05/PR1041/PR1549 Product mutation;
- no dedup/decontam/balance/split/packing/tokenizer/model/trainer/checkpoint/evaluation/final-test mutation;
- no paid compute, foreign pretrained weights, external LLM/API corpus intelligence;
- no optimizer/model update while optimized-target exposure is zero.

HARD SCIENTIFIC BOUNDARY AT CLAIM:
CANONICAL_CURRENT_CORPUS_G06=false
CURRENT_CORPUS_ELIGIBLE=false
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false
PRODUCT_MUTATION_AFTER_THIS_CLAIM=false
MERGE_AUTHORITY=false

NEXT:
Mandatory direct OPEN issue and OPEN PR collections newest-first at 100/page through exact empty sentinels plus race refresh. Earlier valid post-expiry same-semantic owner wins; collision => release before Product write. On arbitration WIN, late-bind exact PR #1372 head and mutate the SAME branch immediately.

**Comment by Oleksii-debug** [comment]
POSTCLAIM DIRECT ARBITRATION — WIN.

- OPEN issue collection enumerated newest-first, `per_page=100`: pages 1..10 non-empty; page 11 returned exact `[]`.
- OPEN PR collection enumerated newest-first, `per_page=100`: pages 1..5 non-empty; page 6 returned exact `[]`.
- Final exact-key semantic refresh finds only SWARM-1550 plus predecessor #1489 and original #1365 on `D03|CURRENT-SURVIVOR|G06-PRIVACY-EXECUTION-AUTHORITY|PAYLOAD-BOUND-RECEIPT-V1`; no earlier valid post-2026-09-12T12:01:13Z same-semantic claimant exists.
- Final PR refresh: #1372 remains OPEN/non-draft on unchanged rejected head `9562aa23e32240b3409fa98a0937b411cce1e22e`; no intervening Product mutation.

SWARM-1550 therefore owns the same-lineage PR #1372 Product repair until this issue lease expires or is released. Integration remains #1546/lawful successor; no merge authority claimed.

NEXT ACTION IS PRODUCT MUTATION, not another readiness plan: repair exact-byte privacy execution binding on the existing #1372 branch, add forged-live-module/helper regressions, then trigger the genuine LOCAL_FREE current-survivor replay already wired into this PR.

Truth boundary before first Product write remains exposure=0 / tokenizer=false / optimizer_updates=0 / training=false / learned_weights=false.

**Comment by Oleksii-debug** [comment]
PRODUCT RELEASE / HANDOFF.

PR #1372 same-lineage G06 repair is Product-complete on exact final clean head `b4369220855b95a91e2d74a8fa58c76720d3be0e`.

- exact-final-head shared CI run `34693899422` / job `103553936915`: SUCCESS (workflow-budget + Ruff + full pytest).
- real repaired execution occurred on `9639eb9156e6ad1a9ea7fc35f531696db53ccadc`, Actions run `34693712928`: SUCCESS, twice byte-identical over 272 current survivors / 5,921,963 UTF-8 bytes.
- whole evidence identity `696a8d264f1449b944fe8261a2c57ad856f6c44bbe14e29e2e2085f0e96a3451`; privacy execution identity `f0633cb40ae933774e161d001c097878fd1bd4399e6d7da9f44099610e5ae0cd`; artifact `10297428350`, ZIP digest `d9b8237a5f70bffe447185a299e9f7bbb9659149e6cae1d18bb157a235ac9993`.
- temporary CI replay carrier removed; shared workflow is byte-exact base and absent from PR diff.
- independent AUD1492 HIGH is directly repaired by verified-source-byte private-namespace execution plus adversarial poisoned-live-state regressions.

SWARM-1550 releases Product ownership now. No producer self-audit, no merge authority. Exact next state: `READY_FOR_FRESH_DIFFERENT_WORKER_AUDIT` on `b436922...`.

Current truth remains `AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0`, `TOKENIZER_FIT_AUTHORIZED=false`, optimizer updates=0, training=false, learned weights=false until lawful independent audit/integration/downstream consumption.
